fiftyone/core/collections.py

```diff
@@ -9269,9 +9256,94 @@ def values(
             _big_result=_big_result,
             _raw=_raw,
             _field=_field,
+            _lazy=False,
         )
         return self._make_and_aggregate(make, field_or_expr)
 
+    def iter_values(
```

Comment:
Can we do `values(..., generator=True)` instead?

For consistency with `add_samples(...,  generator=True)` and other such methods?